geode/src/test/java/akka/stream/alpakka/geode/docs/javadsl/Animal.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
  */
 
-package akka.stream.alpakka.geode.javadsl;
+package akka.stream.alpakka.geode.docs.javadsl;
```

Comment:
Make it `package docs.javadsl` and `docs.scaladsl`. It keeps the docs from referencing internal code.